solr/solrj/src/java/org/apache/solr/client/solrj/cloud/autoscaling/Clause.java

```diff
@@ -686,7 +686,7 @@ boolean isShardAbsent() {
       for (Row r : session.matrix) {
         computedValueEvaluator.node = r.node;
         SealedClause sealedClause = getSealedClause(computedValueEvaluator);
-        if (!sealedClause.getGlobalTag().isPass(r)) {
+        if (r.isLive() && !sealedClause.getGlobalTag().isPass(r)) {
```

Comment:
I brought this back in from the revert because it looks like an improvement that was unrelated to the default policy.

I'm happy to remove it if y'all think I should.